.gitignore

```diff
@@ -0,0 +1,2 @@
+# XCode user settings
+xcuserdata/
```

Comment:
This depends on whether or not Cocoapods should be used for the demo project, but it seems like a good idea to add `DoubleSliderDemo/Pods` to this gitignore.